### PR TITLE
Internal: Remove build-time vendoring repo since it is now unnecessary

### PR DESCRIPTION
**Pull request by martijnvans**
## Description
Remove a build dependencies vendoring repo that seems to no longer be needed.

## Related issue
b/303427818

## How has this been tested?
To test this, I temporarily overrode the presubmit distros with the release distros and ran a presubmit. The result was passing, except for some unrelated flakes: https://source.cloud.google.com/results/invocations/6a79c36b-6a34-4a5f-af55-3ec0ea6ca17c/targets

## Checklist:
- Unit tests
  - [ ] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [ ] Integration tests do not apply.
  - [ ] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [ ] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [ ] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [ ] This PR bumps the version.

### CODE REVIEW

**Review by sophieyfang — APPROVED** [review]

File: kokoro/config/test/image_lists.gcl [hunk 1]
```diff
@@ -134,7 +134,7 @@ sles15_x86_64 = _distro {
     'opensuse-leap',
     'opensuse-leap-15-4',
   ]
-  presubmit = ['sles-15']
+  presubmit = release
```
Comment (sophieyfang) [inline kokoro/config/test/image_lists.gcl:137]: Is this change intended to be submitted?

Comment (martijnvans) [reply kokoro/config/test/image_lists.gcl:137]: oops, sent https://github.com/GoogleCloudPlatform/ops-agent/pull/1458.
